pkgs/development/libraries/libcork/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, lib
+, pkgconfig
+, check
+}:
+stdenv.mkDerivation rec {
+  pname = "libcork";
+  version = "1.0.0--rc3";
+
+  src = fetchFromGitHub {
+    owner = "dcreager";
+    repo = pname;
+    rev = version;
+    sha256 = "152gqnmr6wfmflf5l6447am4clmg3p69pvy3iw7yhaawjqa797sk";
+  };
+
+  # N.B. We need to create this file, otherwise it tries to use git to
+  # determine the package version, which we do not want.
+  #
+  # N.B. We disable tests by force, since their build is broken.
+  postPatch = ''
+    echo "${version}" > .version-stamp
+    echo "${version}" > .commit-stamp
+    sed -i '/add_subdirectory(tests)/d' ./CMakeLists.txt
+  '';
+
+  nativeBuildInputs = [ cmake pkgconfig ];
+  buildInputs = [ check ];
+
+  doCheck = false;
+
+  postInstall = ''
+    ln -s $out/lib/libcork.so $out/lib/libcork.so.1
+  '';
+
+  meta = with lib; {
```

Comment:
```suggestion
  meta = with stdenv.lib; {
```

Reply:
I don't use `stdenv.lib` because IMHO `lib` has nothing to do with `stdenv`, the fact that `stdenv.lib` is a thing is a design mistake. 

Since it's just a matter of preference, AFAIK, I end up just adding the lib attr separately

Reply:
Trying to avoid "design mistakes" by increasing input variables when it really has no benefit just makes me write more comments.